### ISSUE TITLE
Box2D CMakeLists does not include Defines.hpp

### ISSUE THREAD

**Issue by Megaxela** [louis-langholtz/Box2D#23]
I guess that it has to be added there: [link](https://github.com/louis-langholtz/Box2D/blob/dev/Box2D/Box2D/CMakeLists.txt#L43)

**Comment by louis-langholtz** [comment]
Thank you for pointing this out. I'm looking into doing away with the need for `Defines.hpp` as another solution. Not sure yet which one to go with.

**Comment by louis-langholtz** [comment]
I changed the file you linked so that it would include all top-level `*.hpp` files without having to update it anymore as files come and go.

Were you seeing problems related to this before? If so, are those problems gone now? I ask because I hadn't seen any problems before from the Travis CI builds and didn't even realize this had been a problem though once you pointed it out I did want to correct my oversight.
